src/home/HomePage.tsx

```diff
@@ -17,6 +18,7 @@ const HomePage = () => {
 			<div style={{ backgroundColor: '#4848480a' }}>
 				<HomeServiceAreas />
 			</div>
+			<HomeCookieSnackbar timeToWait={2000} />
```

Comment:
We could load from anywhere on the site via the url path, so the user's starting may not always be on the home page. so we should move this to a more reusable component. let's rename the file to `CookieSnackbar` and move the file from the home file to the components file. let's also move this instance from `HomePage.tsx` to `App.tsx` (think: its behavior on the site should be similar to `Header` or `Footer`)